packages/react-components/react-migration-v0-v9/src/components/List/ListItem/useListItem.tsx

```diff
@@ -70,12 +70,21 @@ export const useListItem_unstable = (
     toggleItem?.(e, value);
   });
 
-  // This will give us the onKeyDown and onKeyUp props for Enter and Space
-  const buttonProps = useARIAButtonProps('div', {
-    onClick: handleClick,
+  // avoid passing empty onClick to prevent items unnecessary being narrated as clickable
+  // see https://github.com/microsoft/fluentui/issues/20658
+  const useClick = !!onClick || isSelectionEnabled;
```

Comment:
the `use` prefix is used in React world to signify a hook, this might potentially cause some confusion or even linter issues. I'd suggest going with something like `passClickHandler`.